datafusion/src/execution/context.rs

```diff
@@ -1305,7 +1314,6 @@ mod tests {
     #[tokio::test]
     async fn aggregate_grouped() -> Result<()> {
         let results = execute("SELECT c1, SUM(c2) FROM test GROUP BY c1", 4).await?;
-        assert_eq!(results.len(), 1);
```

Comment:
this is a good change (to remove the redundant assertion)

Reply:
Also necessary to fix the tests - as the number of batches now also depend on the provided concurrency.